src/scenes/Jarjestajat/Jarjestaja/Hakemukset/Muutospyynto/components/MuutospyyntoWizardOpiskelijavuodet.js

```diff
@@ -7,14 +7,13 @@ import Lomake from "../../../../../../components/02-organisms/Lomake";
 import { useChangeObjects } from "../../../../../../stores/changeObjects";
 import { useIntl } from "react-intl";
 
-const getArvoFromKohdeArray = (tyyppi, kohde) => {
+const getKohde = (tyyppi, kohteet) =>
+    R.find(obj => obj.tyyppi === tyyppi)(kohteet || []) || {};
```

Comment:
R.find(obj => obj.tyyppi === tyyppi, kohteet) || {}; would be enough if kohteet had a default value as described in the commend above